t/web-connect.t

```diff
@@ -26,15 +29,18 @@ $t->get_ok($url)->status_is(200)->text_is('title', 'Unable to connect with Githu
 # connect with github
 $url->query->remove('error');
 $url->query->param(code => 42);
-$t->get_ok($url)->status_is(302)->header_is(Location => '/user/profile');
+$t->get_ok($url)->status_is(302);
+
+{
```

Comment:
No. you only need ```SKIP``` labels, when skipping. the only reason why you put ```$TODO``` in a scope is to avoid having to do ```$TODO=undef``` later